packages/fury-adapter-oas3-parser/test/integration/fixtures/petstore.json

```diff
@@ -107,11 +107,10 @@
                               "content": {
                                 "key": {
                                   "element": "string",
-                                  "content": "Content-Type"
+                                  "content": "x-next"
```

Comment:
This seems to be disclosing a bug where the headers object is overriding the `Content-Type`. I think the behaviour should be that the `Content-Type` should still be present but we should only add it if there isn't a user defined Content-Type header.

Reply:
Fixed